test/backward_compatibility/check_backward_compatibility.py

```diff
@@ -83,6 +83,8 @@
     ("aten::empty_meta", datetime.date(2021, 4, 1)),
     ("aten::div", datetime.date(2021, 4, 28)),
     ("aten::divide", datetime.date(2021, 4, 28)),
+    ("aten::_var", datetime.date(2021, 4, 28)),
```

Comment:
We shouldn't have backward compatibility issues since we are no longer modifying the existing signatures.

Reply:
Doesn't the CI check fail when you remove a signature?